schemas/service.js

```diff
@@ -39,10 +40,13 @@ const serviceDescriptor = {
       // FIXME: this will be validated on some const TBD, possibly
       // SeeAlsoService
       type: 'string',
+      minLength: 2,
+      maxLength: cfg.maxLength
     },
     serviceEndpoint: {
       // TODO: can this be improved? pattern starting with https://?
       type: 'string',
+      maxLength: cfg.maxLength,
```

Comment:
Service IDs and Service Descriptors aren't supported on Veres One, so maybe just try to remove them here. Also, we can't use configurable values for these validators -- they have to be hard coded.

Reply:
I removed the config maxLength and made a constant. Removing service and service descriptions is a little more involved.